docs: enable section indexes (#11371)
PrairieLearn/PrairieLearn: docs/developer-guide/index.md

# Developer Guide

> This section provides documentation for all aspects of using PrairieLearn as a developer.

Reviewer: ```suggestion
This section provides documentation for working with the PrairieLearn codebase.
```

> If you are developing your own elements, we recommend running with [local source](../installingLocal.md) in docker. Then check out the [element development guide](../devElements.md).

Reviewer: ```suggestion
If you are developing your own elements, we recommend running with [local source](../installingLocal.md) in Docker. Then check out the [element development guide](../devElements.md).
```
